pkg/trafficpolicy/ingress_types.go

```diff
@@ -15,3 +21,8 @@ type IngressTrafficMatch struct {
 	ServerNames              []string
 	SkipClientCertValidation bool
 }
+
+// GetIngressTrafficMatchName generates the traffic match name
+func GetIngressTrafficMatchName(svc types.NamespacedName, port uint16, protocol string) string {
```

Comment:
It would make sense to implement this type on the MeshService object since the ingress traffic match is applicable to a MeshService. This would make it consistent with how the other traffic match names for the MeshService object are retrieved:
https://github.com/openservicemesh/osm/blob/fc638c334b607cc9f3987e75812c964df208e2f7/pkg/service/types.go#L69-L76

Sorry I might have incorrectly suggested to add this helper in pkg/trafficpolicy earlier.